include/kanread.h

```diff
@@ -0,0 +1,26 @@
+#ifndef KANREAD_H
+#define KANREAD_H
+
+#include "ultra64/ultratypes.h"
+
+/**
+ * s32 Kanji_OffsetFromShiftJIS(s32 sjis);
+ *
+ * Returns the offset of the glyph texture data in the file `kanji` corresponding
+ * to a given 2-byte Shift-JIS codepoint. No range validity check is carried out.
```

Comment:
I think this comment should remain in the .s file only like it currently is. Its no different than how we only keep documentation in C files.

Reply:
ah, wasn't expected it to be different from MM. Is there a way for Intellisense to pick up on the comment in the .s file?

Reply:
ah I suppose intellisense is the reason. I personally dont mind that you have to go to the function for this one, and care more about being consistent across our headers. But if popular opinion is to keep it then keep it